src/ui/CategoryBar/CategoryBar.tsx

```diff
@@ -0,0 +1,61 @@
+'use client';
+import { useState } from 'react';
+import Button from '../shared/Button';
+
+const categoryBarClass = `
+  whitespace-nowrap flex items-center gap-4 px-4 py-2 
+  overflow-x-auto bg-twd-background border-b 
+  border-gray-700 sm:gap-6 sm:px-6  focus:ring-2 focus:ring-twd-secondary-purple
+`;
+
+const CategoryBar: React.FC = () => {
+  const categories = [
+    'All',
+    'Replace',
+    'Distract',
+    'Barrier',
+    'Change status',
+    'Category 6',
+    'Category 7',
+  ];
+
+  const [selectedCategories, setSelectedCategories] = useState<string[]>([]);
+
+  const handleCategoryClick = (category: string) => {
+    // Toggle category selection
+    if (category === 'All') {
+      setSelectedCategories(['All']); // Reset to "All"
+    } else {
+      setSelectedCategories(
+        (prev) =>
+          prev.includes(category)
+            ? prev.filter((c) => c !== category) // Remove if already selected
+            : [...prev.filter((c) => c !== 'All'), category] // Add new category, deselect "All"
```

Comment:
Do we need these comments? Suggest removing as they seem redundant
```suggestion
    if (category === 'All') {
      setSelectedCategories(['All']);
    } else {
      setSelectedCategories(
        (prev) =>
          prev.includes(category)
            ? prev.filter((c) => c !== category)
            : [...prev.filter((c) => c !== 'All'), category]
```